crates/testlang/outputs/cargo/tests/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+#![cfg(test)]
+
+use {anyhow as _, infra_utils as _, semver as _, strum_macros as _, testlang_testing_utils as _};
```

Comment:
I wonder if these imports are needed? is it possible to just remove the deps from the new crate?

Reply:
Done